specs/start_screen/01_main.rst

```diff
@@ -0,0 +1,30 @@
++--------+-----------------+
+| issue  | title           |
++--------+-----------------+
+| 86_    | Start screen    |
++--------+-----------------+
+
+.. _86: https://github.com/gnosis/safe/issues/86
+
+Start screen
+============
+
+1. `Main`_
+2. `User Interface`_
+
+.. _Main:
+
+.. contents:: Table of Contents
+    :depth: 3
+
+1. Problem Definition
+---------------------
+
+- We have update our Safe messaging and slogan a while ago
+  ("Enter the decentralized web").
+- We need to update that inside the app
+
+Inputs, outputs and use case scenarios are omitted since they are trivial.
+Please check `User Interface`_ for the screens.
+
+.. _`User Interface`: 02_user_interface.rst
```

Comment:
would suggest to name 02_user_interface_android.rst

Reply:
I don't know about this. The file contains iOS, too. 
If we have the user interfaces in 1 file, then we can clearly see the differences. They are clients for the same product after all.

Reply:
we could have a separate doc for comparison, I was thinking that for specification, it is better to have different docs per product, because different teams are working on them, and there might be changes to android doc, but iOS doesn't have to be notified about it.

This is just my feeling, but I don't know if it is the best way.

Reply:
I'd like to look at iOS and Android as different clients for the same product. And it's also pretty likely that changes in 1 client affect the other client as well.

@rmeissner @fmrsabino what do you think about this?

Reply:
I think having it in one file can be better for comparison. There should only be smaller differences between the two.

Reply:
Regarding this discussion please also comment this in https://github.com/gnosis/safe/pull/97

Reply:
ok, will merge into one doc